### DISCUSSION TITLE
Does afreeRDP support remote opening of Windows 11 applications on Android operating system

### DISCUSSION THREAD

**Discussion by PYlys** [FreeRDP/FreeRDP#9816] [Q&A]
Does afreeRDP support remote opening of Windows 11 applications on Android operating system

**Comment by akallabeth** [comment]
* If you mean `is windows 11 supported as server target` then yes
* If you are asking about `remote APP/RAILS mode` [[MS-RDPERP]](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdperp/485e6f6d-2401-4a9c-9330-46454f0c5aba) support then no, not implemented on android (pull requests welcome)
